Tests/SwiftModRulesTests/DefaultAccessLevelRuleTests.swift

```diff
@@ -214,6 +214,18 @@ final class DefaultAccessLevelRuleTests: XCTestCase {
         )
     }
 
+    func testClassOpenNotTriggered() throws {
```

Comment:
I'd like to add this test if it's ok.
(I forgot to replace `openKeyward` originally, producing the wrong output like `open open class Class` for this pattern.)